saucectl/examples/.sauce/config.yml

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
@nikolay-advolodkin 

This is already in the `puppeteer` and `playwright` folder, this will not add extra value imho and can be removed

Reply:
@wswebcreation i'm taking a look at this and I'm a bit confused because isn't saucectl a different solution from `puppeteer` and `playwright`?

Reply:
Yes and no, it holds both in the folder. SauceCTL is now called the toolkit and it can run both projects. So that project is even in the correct structure I think 😉

Reply:
I don't see a toolkit example in the repo @wswebcreation , where is it located?